src/gmp/commands/users.js

```diff
@@ -250,9 +255,12 @@ export class UserCommand extends EntityCommand {
 
   saveSettings(data) {
     log.debug('Saving settings', data);
+
     return this.httpPost({
       cmd: 'save_my_settings',
       text: data.timezone,
+      [PARAM_KEYS.DATE]: data.dateFormat,
+      [PARAM_KEYS.TIME]: data.timeFormat,
```

Comment:
I think you may want to update the values here
```suggestion
      [PARAM_KEYS.DATE]: data.userInterfaceDateFormat,
      [PARAM_KEYS.TIME]: data.userInterfaceTimeFormat,
```

Reply:
Good catch!